src/PyodideWorker.js

```diff
@@ -249,19 +254,69 @@ const PyodideWorker = () => {
           pyodidePackage = pyodide.pyimport("matplotlib");
         } catch (_) {}
         if (pyodidePackage) {
+          pyodide.runPython(`
+            import matplotlib.pyplot as plt
+            import io
+            import basthon
+
+            def show_chart():
+                bytes_io = io.BytesIO()
+                plt.savefig(bytes_io, format='jpg')
+                bytes_io.seek(0)
+                basthon.kernel.display_event({ "display_type": "matplotlib", "content": bytes_io.read() })
+            plt.show = show_chart
+            `);
           return;
         }
       },
+      after: () => {},
+    },
+    seaborn: {
```

Comment:
I think we're quickly getting to a point here where sticking these package shim things in separate modules might be preferable

Reply:
Thanks Dan, Ive created another ticket to look into this 👍